xml/cl.xml

```diff
@@ -834,6 +835,12 @@ server's OpenCL/api-docs repository.
         <enum bitpos="31"           name="CL_QUEUE_THREAD_LOCAL_EXEC_ENABLE_INTEL"/>
     </enums>
 
+    <enums name="cl_context_memory_initialize_khr" vendor="Khronos" type="bitmask">
+        <enum bitpos="0"            name="CL_CONTEXT_MEMORY_INITIALIZE_LOCAL_KHR"/>
+        <enum bitpos="1"            name="CL_CONTEXT_MEMORY_INITIALIZE_PRIVATE_KHR"/>
+            <unused start="1" end="31"/>
```

Comment:
```suggestion
            <unused start="2" end="31"/>
```
Also maybe comment here to never extend this bitfield to the upper 32 bits?

Reply:
Fixed - I added the comment too.